frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -76,10 +76,19 @@ export const reloadCard = createThunkAction(RELOAD_CARD, () => {
       Questions.actions.fetch({ id: outdatedQuestion.id() }, { reload: true }),
     );
     const card = Questions.HACK_getObjectFromAction(action);
+
+    const parameterValues = outdatedQuestion.parameters().reduce(
```

Comment:
❤️

Reply:
it would be nice to have a comment that we need to get the same format as `_parameters` have